coderd/coderdtest/uuids_test.go

```diff
@@ -0,0 +1,33 @@
+package coderdtest_test
+
+import (
+	"github.com/google/uuid"
+
+	"github.com/coder/coder/v2/coderd/coderdtest"
+)
+
+func ExampleNewDeterministicUUIDGenerator() {
+	det := coderdtest.NewDeterministicUUIDGenerator()
+	testCases := []struct {
+		CreateUsers []uuid.UUID
+		ExpectedIDs []uuid.UUID
+	}{
+		{
+			CreateUsers: []uuid.UUID{
+				det.ID("player1"),
+				det.ID("player2"),
+			},
+			ExpectedIDs: []uuid.UUID{
+				det.ID("player1"),
+				det.ID("player2"),
+			},
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+		_ = tc
+		// Do the test with CreateUsers as the setup, and the expected IDs
+		// will match.
+	}
```

Comment:
This doesn't seem to be running the test cases?

Reply:
:facepalm: 

I had this in another local branch at first. I'll bring in the correct code, good catch.

Reply:
This is a [TestableExample](https://go.dev/blog/examples). Looks like I kinda gave up here. I'm going to just remove. I have example usage in the idp package now.